localization.c

```diff
@@ -189,7 +189,7 @@ LocalizedTime(const time_t t, LPTSTR buf, size_t size)
     /* Convert Unix timestamp to Win32 SYSTEMTIME */
     FILETIME lft;
     SYSTEMTIME st;
-    LONGLONG tmp = Int32x32To64(t, 10000000) + 116444736000000000;
+    LONGLONG tmp = ((LONGLONG)t * 10000000) + 116444736000000000;
```

Comment:
Looks good, but isn't it cleaner to write this as t * 10000000LL + ...  without the cast?

Reply:
Works for me! I'll update the PR.